app/docs/0.10.x/loadbalancing.md

```diff
@@ -182,6 +184,116 @@ as a single target, with the specified weight. Upon every proxied request
 to this target it will query the nameserver again.
 
 
-[Back to TOC](#table-of-contents)
+[Back to Table of Contents](#table-of-contents)
+
+### **Blue-green deployments**
```

Comment:
as this is a title, i feel it should be capitalized; we should probably update the rest of this doc to match